src/templates/fork_observer_config.toml

```diff
@@ -0,0 +1,30 @@
+# fork-observer base configuration file
+
+# Database path of the key value store. Will be created if non-existing.
+database_path = "db"
+
+# path to the location of the static www files
+www_path = "./www"
+
+# Interval for checking for new blocks
+query_interval = 1
```

Comment:
I have this a 10 seconds for my nodes. Every second might be overkill. Maybe do 5s intervals?